cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,47 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
+import UserPageObject from '../support/pages/user.pageObject';
+
+const faker = require('faker');
+const userPage = new UserPageObject();
+
 describe('User', () => {
+  let user;
+
+  const followedUser = {
+    username: faker.name.firstName(),
+    bio: faker.lorem.words(),
+    email: faker.internet.email(),
+    password: faker.internet.password(),
+  };
+
   before(() => {
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
+  });
+
+  beforeEach(() => {
+    cy.task('db:clear');
+    homePage.visit();
 
+    cy.register(followedUser.email, followedUser.username, followedUser.password);
+    cy.login(user.email, user.username, user.password);
   });
 
   it.skip('should be able to follow the another user', () => {
+    userPage.visitFollowedUser(followedUser.username);
+
+    userPage.clickFollowBtn();
+    userPage.assertUnfollowBtn(followedUser.username);
+  });
+  it.skip('should be able to unfollow the another user', () => {
+    userPage.visitFollowedUser(followedUser.username);
+    userPage.clickFollowBtn();
```

Comment:
You are clicking on the 'Follow' button and asserting the 'Unfollow' button in both test cases. The second test case should start with an assertion of the 'Unfollow' button (since the user has been followed in the first test case), then click on 'Unfollow', and finally assert the 'Follow' button.